src/style.css

```diff
@@ -1,3 +1,92 @@
+/* resetting settings */
+
 body {
   margin: 0;
 }
+
+ul,
+ol,
+li {
+  list-style: none;
+
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+/* header */
+[class*="__container"] {
+  padding: 0 50px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
```suggestion

.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
```

Reply:
Ok, i deleted redundant classes, and change some settings, can you check please.